src/server/templates/start.njk

```diff
@@ -1,8 +1,10 @@
 {% extends 'layouts/base.njk' %}
 
+{% set serviceTitle = '' %}
+
```

Comment:
Clears the title in the header next to the NHS logo. On the start page it is intentionally removed because we use the name clearly in the blue header. @Demotive can probably give you more background and rationale behind it.